src/completion/completionengine.vala

```diff
@@ -105,12 +105,11 @@ namespace Vls.CompletionEngine {
     }
 
     void finish (Jsonrpc.Client client, Variant id, Collection<CompletionItem> completions) {
-        var json_array = new Json.Array ();
```

Comment:
To keep the purpose of this merge commit clear, please don't change anything else that is not strictly pertinent to the issue.